.scalafmt.conf

```diff
@@ -4,3 +4,5 @@ assumeStandardLibraryStripMargin = true
 danglingParentheses = true
 rewrite.rules = [AvoidInfix, SortImports, RedundantBraces, RedundantParens, SortModifiers]
 docstrings = JavaDoc
+align.openParenCallSite = true
+align.openParenDefnSite = true
```

Comment:
Can we split the formatting changes into a separate PR? There are two separate changes that both could be controversial in this PR and it might be simpler to not have the reviews/comments of the two intertwined.

Reply:
Sorry I should've noted that these two settings were default to be `true` in scalafmt 1.5, which is the version used before this PR. In scalafmt 1.6 the defaults were changed to `false`. So I added them here to avoid more changes than necessary to this PR. If we want to change to sue 1.6's default, we can open a new PR to remove these two settings.

Reply:
Damn my bad I tried to anticipate 1.6 by adding `danglingParentheses = true` but I missed those ones.
So you can remove `danglingParentheses = true` then since in 1.6 it's the default.

Reply:
@kailuowang ah okay I see that makes sense. 👍